blog/2024-09-18-case-study-falai/index.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: case-study-falai
+title: How fal.ai offers the fastest generative ai in the world
+description: >
+  Exploring Tigris' shadow bucket feature for fun and thumbnails. In the blog
+  post the feature will be used in an less conventional fashion. An additional
+  elixir service will generate thumbnails, where source files as well as
+  generated thumbnails are stored on Tigris, while making use of the included
+  CDN features of Tigris.
+image: ./falai.jpg
+keywords:
+  [
+    object storage,
+    blob storage,
+    s3,
+    ai,
+    generative ai,
+    stable diffusion,
+    flux,
+    diffusion,
+    performance,
+    cost,
+  ]
+authors: [ks]
+tags: [object storage, s3, ai, case study]
+---
+
+# How fal.ai offers the fastest generative ai in the world
+
+_We’re Tigris, a globally distributed object storage platform built for
+high-performance workloads. It’s designed to work seamlessly across clouds,
+countries, and continents, making it simple and efficient for developers.
+[Try it out, you’ll be up and running in a minute.](https://storage.new/)_
+
+<!-- truncate -->
```

Comment:
Move this below the `span` tag just above the next headline. Docusaurus shows the preview of the blog post on the homepage up to the point where you add `<!-- truncate -->`.